Update New-CsTeamsMessagingPolicy.md (#11362)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/New-CsTeamsMessagingPolicy.md

Accept wildcard characters: False
```

### -CreateCustomEmojis

> This setting enables creation of custom emojis/reactions within a tenant for the specified policy users.

Reviewer: ```suggestion
This setting enables the creation of custom emojis and reactions within an organization for the specified policy users.
```

```yaml
Type: Boolean
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -DeleteCustomEmojis

> This settings enable/disable the editing/deletion of custom emojis and reactions for the users included in the messaging policy

Reviewer: ```suggestion
These settings enable and disable the editing and deletion of custom emojis and reactions for the users included in the messaging policy.
```
